app/src/main/res/layout/copy_share_buttons_layout.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="center"
+    android:orientation="horizontal"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
```

Comment:
There is no constraint layout here

Reply:
I found midway trough it that when you have match_parent in width for example you don't need to put constraints, should we do this for the rest or from now on, or do you think it's safe to always have those?